devops/app_management/ansible.rst

```diff
@@ -362,6 +364,8 @@ Application properties can be set via *application_properties* variable:
     ``'true'``).
 
 
+.. _ansible-add-route:
+
```

Comment:
Didn't realize before but the documentation is missing for configuring reCaptcha when a new domain is added. Guess we should add a step to the instructions (line 396).

I'd probably just tell people to register the domain with the default key and link to the migration information if anyone needs more details. I'll leave the details up to you though.